### PR TITLE
split Az.Automation exeamples

### PR DESCRIPTION
**Pull request by v-yuzhichen**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Automation/Automation/help/New-AzAutomationSoftwareUpdateConfiguration.md [hunk 1]
```diff
@@ -45,13 +45,19 @@ Creates a software update configuration that runs on a schedule to update a list
 Creates a software update configuration to install critical updates on two Windows azure virtual machines once every Saturday 9PM. Update duration is set to 2 hours in this example.
 
 ```powershell
-PS C:\> $startTime = [DateTimeOffset]"2018-09-13T21:00"
-PS C:\> $targetMachines = @( `
+$startTime = [DateTimeOffset]"2018-09-13T21:00"
+```
```
Comment (ziyuezh576) [inline src/Automation/Automation/help/New-AzAutomationSoftwareUpdateConfiguration.md:49]: same as the first comment

File: src/Automation/Automation/help/New-AzAutomationSchedule.md [hunk 2]
```diff
@@ -66,29 +66,37 @@ The **New-AzAutomationSchedule** cmdlet creates a schedule in Azure Automation.
...
 > Note: Schedule *StartTime* is calculated by adding the *TimeZone* Offset to provided *StartTime*
 
 ### Example 3: Create a recurring schedule
+```powershell
+$StartTime = Get-Date "13:00:00"
 ```
-PS C:\> $StartTime = Get-Date "13:00:00"
-PS C:\> $EndTime = $StartTime.AddYears(1)
-PS C:\> New-AzAutomationSchedule -AutomationAccountName "Contoso17" -Name "Schedule02" -StartTime $StartTime -ExpiryTime $EndTime -DayInterval 1 -ResourceGroupName "ResourceGroup01"
+```powershell
```
Comment (ziyuezh576) [inline src/Automation/Automation/help/New-AzAutomationSchedule.md:95]: same as the first comment

File: src/Automation/Automation/help/New-AzAutomationSchedule.md [hunk 3]
```diff
@@ -66,29 +66,37 @@ The **New-AzAutomationSchedule** cmdlet creates a schedule in Azure Automation.
...
 The first command gets the time zone ID from the system and stores it in the $TimeZone variable.
 The second command creates a schedule that runs one time on the current date at 11:00 PM in the specified time zone.
 
 ### Example 2: Create a one-time schedule in another time zone
+```powershell
+$TimeZone = "Europe/Paris"
 ```
-PS C:\> $TimeZone = "Europe/Paris"
-PS C:\> New-AzAutomationSchedule -AutomationAccountName "Contoso17" -Name "Schedule01" -StartTime "23:00Z" -OneTime -ResourceGroupName "ResourceGroup01" -TimeZone $TimeZone
+```powershell
```
Comment (ziyuezh576) [inline src/Automation/Automation/help/New-AzAutomationSchedule.md:83]: same as the first comment

File: src/Automation/Automation/help/New-AzAutomationSchedule.md [hunk 4]
```diff
@@ -66,29 +66,37 @@ The **New-AzAutomationSchedule** cmdlet creates a schedule in Azure Automation.
 ## EXAMPLES
 
 ### Example 1: Create a one-time schedule in local time
+```powershell
+$TimeZone = ([System.TimeZoneInfo]::Local).Id
 ```
-PS C:\> $TimeZone = ([System.TimeZoneInfo]::Local).Id
-PS C:\> New-AzAutomationSchedule -AutomationAccountName "Contoso17" -Name "Schedule01" -StartTime "23:00" -OneTime -ResourceGroupName "ResourceGroup01" -TimeZone $TimeZone
+```powershell
```
Comment (ziyuezh576) [inline src/Automation/Automation/help/New-AzAutomationSchedule.md:72]: same as the first comment

File: src/Automation/Automation/help/New-AzAutomationSoftwareUpdateConfiguration.md [hunk 5]
```diff
@@ -45,13 +45,19 @@ Creates a software update configuration that runs on a schedule to update a list
...
     "/subscriptions/22e2445a-0984-4fa5-86a4-0280d76c4b2c/resourceGroups/compute/providers/Microsoft.Compute/virtualMachines/vm-w-01", `
     "/subscriptions/22e2445a-0984-4fa5-86a4-0280d76c4b2c/resourceGroups/compute/providers/Microsoft.Compute/virtualMachines/vm-w-02"
     )
-PS C:\> $duration = New-TimeSpan -Hours 2
-PS C:\> $schedule = New-AzAutomationSchedule -ResourceGroupName "mygroup" `
+```
+```powershell
+$duration = New-TimeSpan -Hours 2
+```
+```powershell
```
Comment (ziyuezh576) [inline src/Automation/Automation/help/New-AzAutomationSoftwareUpdateConfiguration.md:59]: same as the first comment

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

**Review by ziyuezh576 — APPROVED** [review]

**Review by ziyuezh576 — APPROVED** [review]

File: src/Automation/Automation/help/Get-AzAutomationSourceControlSyncJobOutput.md [hunk 6]
```diff
@@ -28,12 +28,13 @@ This command gets the output of source control sync job with id 08d6d266-27b6-46
...
 ```powershell
-PS C:\> Get-AzAutomationSourceControlSyncJobOutput -ResourceGroupName "rg1" `
+Get-AzAutomationSourceControlSyncJobOutput -ResourceGroupName "rg1" `
                                                         -AutomationAccountName "devAccount" `
                                                         -Name "VSTSNative"
                                                         -Id "08d6d266-27b6-463c-beea-bc48a67ace15" `
                                                         -Stream Output | ForEach-Object {$_.summary}
-
+```
+```output
```
Comment (BethanyZhou) [inline src/Automation/Automation/help/Get-AzAutomationSourceControlSyncJobOutput.md:37]: There is missing \```

**Review by BethanyZhou — COMMENTED** [review]
Please check all examples

**Comment by BethanyZhou** [comment]
please resolve these conflicts. @v-yuzhichen

**Review by ziyuezh576 — APPROVED** [review]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**